exporters/otlp/otlplog/otlploggrpc/client.go

```diff
@@ -133,26 +159,40 @@ func (c *client) UploadLogs(ctx context.Context, rl []*logpb.ResourceLogs) error
 	defer cancel()
 
 	return c.requestFunc(ctx, func(ctx context.Context) error {
+		trackExportFunc := c.trackExport(context.Background(), int64(len(rl)))
```

Comment:
Apologies for the omission here. We need to move the relevant reporting logic out of `requestFunc`.

`requestFunc` has a retry mechanism at the underlying level, which may ultimately result in duplicated data reporting.